lib/bugsnag.js

```diff
@@ -101,6 +101,18 @@ module.exports = Bugsnag = (function() {
     }, autoNotifyCallback(err));
   };
 
+  Bugsnag.koaHandler = function(err, ctx) {
+    var request;
+    Configuration.logger.info("Handling koa error: " + (err.stack || err));
+    request = ctx.req;
+    request.protocol = ctx.request.protocol;
```

Comment:
@snmaynard Did you see this part?
If you directly use bugsnag to notify the error, it doesn't handle protocol/host correctly in Koa.